CHANGES.rst

```diff
@@ -8,6 +8,7 @@ There's a frood who really knows where his towel is.
 
 - The 'drop' event and its 'target' method are used to obtain the element (column) and not generate ambiguity (fixes `#861 <https://github.com/collective/collective.cover/issues/861>`_). [Mubra]
 
+- A "row" object is generated and its location in the DOM is added with the even "drop.target" (fixes `#868 <https://github.com/collective/collective.cover/issues/868>`_). [Mubra]
```

Comment:
Please new entries at the top.
User id must be in a new line.
see:

https://docs.plone.org/develop/styleguide/documentation.html#tracking-changes

Reply:
@idgserpro  sorry, I correct it shortly